QUEST_LV_0100.tsv

```diff
@@ -8273,11 +8273,11 @@ QUEST_LV_0100_20151001_008268	That fellow knows well about the purifiers in the
 QUEST_LV_0100_20151001_008269	{memo X}You want to learn my skills?{nl}It's not hard for me to teach you, but would you be able to handle it?
 QUEST_LV_0100_20151001_008270	The Goddess always told us to help those around us.{nl}By helping others near you, you will help your spirit as well.
 QUEST_LV_0100_20151001_008271	Great King Zachariel's ordeal will not be easy.{nl}With the skill you have, he should have acknowledged you.
-QUEST_LV_0100_20151001_008272	The kingdom has lost all its hope. {nl}It does not have any will to stand back anymore and is just crying out for the Goddess.{nl}
-QUEST_LV_0100_20151001_008273	It is during this time that the people came forward saying they dreamed of the Goddess. {nl}It is you, the Revelator. {nl}
+QUEST_LV_0100_20151001_008272	Our kingdom has lost all its hope. {nl}It does not have any will to stand back anymore and is just crying out for the Goddesses.{nl}
+QUEST_LV_0100_20151001_008273	And from such chaotic times, there are now people who come forward stating that they've dreamed of the Goddess. {nl}People such as yourself are the Revelators. {nl}
```

Comment:
Alrighty, I'll just take care of the merge conflict when it comes. xD Your's sound better